packages/ui-flex/src/Flex/styles.ts

```diff
@@ -85,6 +86,11 @@ const generateStyle = (
     'row-reverse': 'row-reverse'
   }
 
+  // gap css prop
+  const getGapValue = (gap: FlexProps['gap'], theme: FlexTheme) => {
```

Comment:
If there is only a return in the arrow function, leave it out:

`const getGapValue = (gap: FlexProps['gap'], theme: FlexTheme) => getShorthandPropValue('Flex', theme, gap, 'gap')`

More consistent and easier to read